components/Card.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="grid grid-cols-3 gap-5 p-5">
+    <UCard v-for="item in paginatedItems" :key="item.id" class="w-[100%]" :ui="{
+      header: {
+        base: '',
+        background: '',
+        padding: '',
+      },
+    }">
+      <template #header>
```

Comment:
you are rendering multiple card in cards, that's a bad convention.
ideally these looping and data fetching should on page level.
this component should be responsible to show and handle the card specific stuffs.

Reply:
Have resolved this comment